StreamDeckSDK/ESDAction.cpp

```diff
@@ -39,6 +39,30 @@ void ESDAction::KeyDown(const nlohmann::json& settings) {
 void ESDAction::KeyUp(const nlohmann::json& settings) {
 }
 
+void ESDAction::DialUp(const nlohmann::json& settings) {
+
+}
+
+void ESDAction::DialDown(const nlohmann::json& settings) {
+
+}
```

Comment:
The naming here's consistent, but a bit confusing for a dial.

What's the physical interaction? Is it a button push, or e.g. a capacitive touch sensor?

If it's a touch sensor, perhaps `TouchStart` and `TouchEnd` instead?

Reply:
It's a button press on the Dial, you can rotate and press them.
There's also a touch screen but I did not start working on it.